src/constants.ts

```diff
@@ -229,3 +229,5 @@ To see this in action:
 export const termsWithImagesAndCountsDescription = `Pass boolean flags for \`displaySearchSuggestionImages\` and \`displaySearchSuggestionResultCounts\` fields to display images and counts for search suggestions. These fields need to be made displayable before they can be used. Please contact your Constructor Integration Engineer for details.`;
 
 export const debounceDescription = `Pass an integer to \`debounce\` to override the recommended, default delay employed for debouncing autocomplete network requests between keystrokes as your users type into the text input field. The default value is 250, which results in a debounce delay of 250 milliseconds.`;
+
+export const translationsDescription = `Pass a \`translations\` object to display translatable words in your preferred language.`;
```

Comment:
Not sure where but can we document these translatable words somewhere? As a use I want to know all of the words I should be passing translations for